community/modules/scheduler/schedmd-slurm-gcp-v5-controller/main.tf

```diff
@@ -36,12 +36,19 @@ locals {
   enable_public_ip_access_config = var.disable_controller_public_ips ? [] : [{ nat_ip = null, network_tier = null }]
   access_config                  = length(var.access_config) == 0 ? local.enable_public_ip_access_config : var.access_config
 
+  instance_image_project_raw = lookup(var.instance_image, "project", "")
+  instance_image = {
+    name    = lookup(var.instance_image, "name", "")
+    family  = lookup(var.instance_image, "family", "")
+    project = strcontains(local.instance_image_project_raw, "/") ? local.instance_image_project_raw : "projects/${local.instance_image_project_raw}/global/images/family"
```

Comment:
I think the long form is only needed when family is used (not name). Have you tested with name? I think you will want to also hide the transformation behind the condition that family is set.

Reply:
I've removed this block altogether after my latest update.